chain/chain/src/chain.rs

```diff
@@ -1489,31 +1488,13 @@ impl Chain {
         me: &Option<AccountId>,
```

Comment:
nit: we might want to rename this function to "process_block_and_unroll_orphans"

Reply:
Not going to do that in this PR.